src/components/expression/expression.js

```diff
@@ -172,6 +204,22 @@ genotet.ExpressionView = function(viewName, params) {
     .on('genotet.updatePanel', function() {
       this.panel.dataLoaded();
     }.bind(this));
+
+  // Set up link callbacks.
+  $(this).on('genotet.addProfile', function(event, data) {
+    var genes = data.id.split(',');
```

Comment:
As we discussed before, link manager should only forward the signal. The data should be retrieved or processed in the target view. Therefore I can only forward the data, but not pass in an array.